dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -712,13 +712,13 @@ private String getCursorValue(final ConnectionContext state, final CursorBasedRe
         return "0";
     }
 
-    private List<OperationResult<ScanResult<String>>> scatterGatherScan(final CursorBasedResult<String> cursor, final String... pattern) {
+    private List<OperationResult<ScanResult<String>>> scatterGatherScan(final CursorBasedResult<String> cursor, final int count, final String... pattern) {
```

Comment:
So we are adding `count` although it was not there before? Are we adding a new API call?